front/package.json

```diff
@@ -110,7 +110,7 @@
     "react-hook-form": "^7.33.1",
```

Comment:
Did you have a chance to look at babel-plugin-react-intl? Maybe we can just delete it? It doesn't seem like we use it anywhere. CC https://github.com/CitizenLabDotCo/citizenlab/pull/2791

Reply:
It is used in the `extract-intl` script and replacing the plugin with the new one breaks it for some reason. Since it works as is and it doesn't really affect the dev process, I suggest we leave it as is for the moment and look into it at a later stage.

Reply:
Where exactly is it used in extract-intl? Doesn't this just use babel-core?

Reply:
Here:
```
const options = { plugins: ['react-intl'], filename: fileName };
```

Didn't we look into this together at some point, actually? Or maybe I'm misremembering. In any case - switching this deleted all the translations and I wouldn't invest time in fixing it with the new plugin atm, as it really doesn't affect us all that much.

Reply:
Just saw there were perf improvements using the new package.

Reply:
But definitely, if you feel it's important, feel free to pick it up in a separate PR. Shouldn't block us here, though, I believe :)